container-storage-setup.sh

```diff
@@ -1094,6 +1100,25 @@ setup_storage() {
     Error "Failed to setup logical volume for $CONTAINER_ROOT_LV_MOUNT_PATH."
     return 1
   fi
+  create_css_config
+}
+
+create_css_config(){
+  local input_filename="input"
+  local output_filename="output"
+  local metadata_filename="metadata"
+  local metadata_dir=${_CONFIG_DIR}/${CONFIG_NAME}
+  mkdir -p ${metadata_dir}
+  cp ${_STORAGE_IN_FILE} ${metadata_dir}/${input_filename}
+  cp ${_STORAGE_OUT_FILE} ${metadata_dir}/${output_filename}
+  cat <<EOF > ${metadata_dir}/${metadata_filename}
+  _M_STORAGE_DRIVER=${STORAGE_DRIVER}
+  _M_CONTAINER_THINPOOL=${CONTAINER_THINPOOL}
+  _M_VG=${VG}
+  _M_CONTAINER_ROOT_LV_NAME=${CONTAINER_ROOT_LV_NAME}
+  _M_CONTAINER_ROOT_LV_MOUNT_PATH=${CONTAINER_ROOT_LV_MOUNT_PATH}
```

Comment:
We probably should store _M_DOCKER_ROOT_VOLUME= information to know if user used old config option to setup extra volume. That will affect cleanup.